developer_guide/installation/system_requirements/setup_requirements_debian8.rst

```diff
@@ -0,0 +1,177 @@
+Setup System Requirements on Debian 8 Jessie
+============================================
+
+Here is a quick guide to setup the :doc:`system_requirements` on Debian 8 Jessie.
+
+.. note::
+
+    Please perform the following commands as root.
+
+System installation
+-------------------
+
+Base dependencies
+*****************
+
+.. code-block:: bash
+    :linenos:
+
+    $ apt-get update
+    $ apt-get install mysql-server apache2 libapache2-mod-php5 php5-cli php5-apcu php5-mcrypt php5-intl php5-mysql php5-curl php5-gd
+    $ php5enmod mcrypt
+    $ a2enmod rewrite
+
+.. note::
+    PHP 5.6 provided in Debian 8 Jessie comes with the Zend OPcache opcode cache. Only the data cache provided by APCu is needed.
+
+Choosing the product storage
+****************************
+
+.. include:: /reference/technical_information/choose_database.rst.inc
+
+Based on this formula, either you need :ref:`installing-mongodb`, either you can directly go to the :ref:`system-configuration` section.
+
+.. _installing-mongodb:
+
+Installing MongoDB
+******************
+
+* Install MongoDB server
+
+.. code-block:: bash
+    :linenos:
+
+    $ apt-get update
+    $ apt-get install mongodb
+
+.. note::
+
+    Akeneo PIM will not work with MongoDB 3.*. *The supported version are 2.4 and 2.6*.
```

Comment:
It may work, if we test before the release we could be able to day it works in "experimental" mode.

Obviously, testing locally all the features is not enough to announce that we support it (like php7, for official support we need dedicated CI builds imho).